style.html

```diff
@@ -4,10 +4,15 @@
 <custom-style>
   <style>
     html {
-      --lumo-border-radius: 0.25em;
+      /* Border radius */
+      --lumo-border-radius-s: 0.25em; /* Checkbox, badge, date-picker year indicator, etc */
+      --lumo-border-radius-m: 0.25em; /* Button, text field, menu overlay, etc */
```

Comment:
BTW is that intentional that `s` and `m` are the same? I haven't seen any mention in the PR description.

Reply:
Yeah, it’s intentional. 

One reason is that the S radius should be used for example for checkbox, and we should avoid any visual breaking changes. I realized, with the Lumo editor, that larger radiuses make the checkbox round very easily, so there should be a separate property for elements that require a smaller radius.